alg9-totalPrice.js

```diff
@@ -0,0 +1,48 @@
+/* Create a function that takes an array of objects (groceries) which calculates the total price and returns it as a number. A grocery object has a product, a quantity and a price, for example:
+{
+  "product": "Milk",
+  "quantity": 1,
+  "price": 1.50
+}
+Examples
+// 1 bottle of milk:
+getTotalPrice([
+  { product: "Milk", quantity: 1, price: 1.50 }
+]) ➞ 1.5
+// 1 bottle of milk & 1 box of cereals:
+getTotalPrice([
+  { product: "Milk", quantity: 1, price: 1.50 },
+  { product: "Cereals", quantity: 1, price: 2.50 }
+]) ➞ 4 */
+
+const itemPrice = obj => {
+  const { quantity, price } = obj;
+  return quantity * price;
+}
+
+const totalPrice = array => {
+  var ttlPrice = 0;
+
+  for (let i = 0; i < array.length; i++) {
+    ttlPrice += itemPrice(array[i]);
+  }
+  return ttlPrice;
+}
```

Comment:
good job! now use Array.proptype.reduce and implement this. This problem is a bit too easy, but it's a great problem to learn how to use reduce.